kubera/dmaas/credentials/minio.go

```diff
@@ -0,0 +1,184 @@
+package minio
+
+import (
+	"encoding/json"
+	"fmt"
+	"os"
+
+	"github.com/go-resty/resty/v2"
+	auth "github.com/mayadata-io/kubera-api-testing/kubera/authentication"
+	"github.com/mayadata-io/kubera-api-testing/kubera/connection"
+	"github.com/tidwall/gjson"
+)
+
+// CloudCredential conaitns all parameters which should be passed to add cloud credential
+/*
+Title: Name with which this credential has been added
+ProjectID: Project ID related to Group ID
+ProviderID: ID of backup provider like 'aws','gcp','minio' or 'cloudian'.
+Cred: Denotes backup provider credential.
+*/
+type CloudCredential struct {
+	Title      string   `json:"name"`
+	ProjectID  string   `json:"projectId"`
+	ProviderID string   `json:"providerId"`
+	Cred       CredType `json:"credential"`
+}
+
+// CredType is cloud provider access id and secret key
+type CredType struct {
+	AccessID  string `json:"access_id"`
+	SecretKey string `json:"secret_key"`
+}
+
+/*
+ACCESS_ID is Env variable contains value of cloud access id.
+ACCESS_ID is Env variable contains value of cloud secret key.
+*/
+const (
+	cloudAccessID  = "ACCESS_ID"
+	cloudSecretKey = "SECRET_KEY"
+)
+
+/*
+Fetch group id .
+One Group will be created in every user account for every Project that he have access to.
+*/
+func getGroupID(token auth.Token) (string, error) {
+	client := resty.New()
+	kErr := auth.KuberaError{}
+	conn := connection.Get()
+	url := conn.HostName + "v3/groups"
+	body := fmt.Sprintf("token=%s;authProvider=localAuthConfig", token.JWT)
+	resp, err := client.R().
+		SetHeader("Accept", "application/json").
+		SetHeader("Cookie", body).
+		SetError(&kErr).
+		Get(url)
```

Comment:
How about following syntax:
```go
Get(fmt.Sprintf("%s/v3/groups", connection.Get().HostName))
```
Does HostName end with `/` ?
We can expose another function in connection package e.g. `connection.GetHostName()`